src/node_quic_default_application.cc

```diff
@@ -143,12 +144,12 @@ bool DefaultApplication::SendStreamData(QuicStream* stream) {
 
   for (;;) {
     Debug(stream, "Starting packet serialization. Remaining? %d", remaining);
-    MallocedBuffer<uint8_t> dest(Session()->GetMaxPacketLength());
+    auto packet = CreateStreamDataPacket();
```

Comment:
It’s easier to read if the type is spelled out ;)

Reply:
Can I totally disagree with that? ;-)

Reply:
You can, but it’s hard to reason about the code locally here if the type isn’t obvious :)